src/content/docs/vulnerability-management/change-vulnerability-status.mdx

```diff
@@ -0,0 +1,203 @@
+
+    <img
+    title="Vulnerabilities tab scoped to a specific entity"
+    alt="An image showing the scoped vulnerabilities tab of Vulnerability Management with filter tab open"
+    src={vmAffectedEntityVulnList}
+    />
+
+1. Choose the vulnerability that you want to update, then click **Change to affected**.
+
+    <img
+        title="Vulnerabilities tab scoped to a specific entity and only ignored instances"
+        alt="An image showing the scoped vulnerabilities tab of Vulnerability Management filtered to only ignored instances"
+        src={vmAffectedEntityFiltered}
+    />
+
+1. Fill out the form to explain the reasoning behind changing this vulnerability to Affected.
+        
+    <img
+        title="Vulnerability instance details form"
+        alt="An image showing the affected form on vulnerability instance details page"
+        src={vmAffectedEntityForm}
+    />
+
+    </Collapser>
+    <Collapser
+        id="bulk-change-status"
+        title="Remove Ignored status across multiple instances"
+    >
+
+1. Go to **[one.newrelic.com](https://one.newrelic.com/) > All capabilities > Vulnerability Management** > Click the **Vulnerabilities** tab 
+1. From the settings cog, select the **Ignored** checkbox. This adds an ignore column to your vulnerability table.
+1. Select a vulnerability with an ignored count greater than 0. 
+
+    <img
+        title="Vulnerabilities tab with ignored column added"
+        alt="An image showing the all entities vulnerabilities tab with an ignore column added"
+        src={vmAffectedBulkVulnList}
+    />
+
+You can also bulk manage your Ignored vulnerabilities with the filter bar. This method is better if you X:
+
+1. Update the filter from `Status = Affected` to `Status = Ignored`. 
+
+    <img
+        title="Vulnerabilities details"
+        alt="An image showing the vulnerability details page with the status dropdown open"
+        src={vmAffectedBulkDetails}
+    />
+
+1. Referring to the **Area of Impact** section, select the entities you want to mark Affected.
+
+    <img
+        title="Vulnerabilities details with instances selected"
+        alt="An image showing the vulnerability details page with affected instances selected"
+        src={vmAffectedBulkSelection}
+    />
+
+1. Fill out the form to explain the reasoning behind changing this vulnerability to Affected.
+
+    <img
+        title="Vulnerabilities details with form open"
+        alt="An image showing the vulnerability details page with form to change instances to affected"
+        src={vmAffectedBulkForm}
+    />
+
+    </Collapser>
+</CollapserGroup>
+
+## Surface Ignored vulnerabilities [#surface-vulnerabilities]
+
+Before you assign **Ignored** to a vulnerability or entity, it's important to understand that the change affects how other code owners manage that particular instance. For example, marking a vulnerability or entity as **Ignored** prevents other users with account access from seeing that vulnerability. 
```

Comment:
Let's get consistent: Should we put things like Ignored and Affected in ** or code backticks? Whatever we do, we should it every time